### PR TITLE
Minor enhancements to global media

### PR DESCRIPTION
**Pull request by roborourke**
Better variable name and a shorter default media library name.

### CODE REVIEW

File: inc/global_assets/namespace.php [hunk 1]
```diff
@@ -108,25 +108,10 @@ function configure_site() {
...
-	$provider_name = $name;
-
-	$site_name = get_blog_option( Global_Content\get_site_id(), 'blogname' );
-	if ( ! empty( $site_name ) ) {
-		$provider_name = sprintf( '%s %s', $site_name, __( 'Media' ) );
-	}
-
-	return $provider_name;
+function set_global_site_provider_name() : string {
+	return __( 'Global Media Library', 'altis' );
```
Comment (jazzsequence) [inline inc/global_assets/namespace.php:114]: With the reduced scope of this function, it feels like this value should be filterable if people want to change the name.

**Review by jazzsequence — APPROVED** [review]
Looks good. Just one comment about possibly adding a filter for the global media library site name.

File: inc/global_assets/namespace.php (hunk 1 above)
Comment (roborourke) [reply inc/global_assets/namespace.php:114]: It can be filtered upstream in the AMF WP plugin - that's what this is hooking into :)

Comment (jazzsequence) [reply inc/global_assets/namespace.php:114]: Okay, maybe I'm remembering that I saw that already. In that case the actual site name is irrelevant and this is good. 👍

**Pull request merged**